tests/system/robot/chromeTests.py

```diff
@@ -69,7 +69,7 @@ def test_aria_details():
 	actualSpeech = _chrome.getSpeechAfterKey('downArrow')
 	_asserts.strings_match(
 		actualSpeech,
-		"The word  marked content  cat  out of marked content  has a comment tied to it."
+		"The word  marked content  has details  cat  out of marked content  has a comment tied to it."

```

Comment:
should this be reported after or before "cat"?

Reply:
The "has details" should be outside (i.e. reported before or after) the marked content, so the scope of the mark is clear.

Reply:
I think this is fine. Personally I think that "marked content has details" reads better than "has details marked content". Might be good to get a few more opinions.